batch/batch.go

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package batch
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
+	"github.com/Azure/azure-sdk-for-go/services/batch/2017-09-01.6.0/batch"
+	batchARM "github.com/Azure/azure-sdk-for-go/services/batch/mgmt/2017-09-01/batch"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/to"
+	uuid "github.com/satori/go.uuid"
+)
+
+const (
+	stdoutFile string = "stdout.txt"
+	stderrFile string = "stderr.txt"
+)
+
+// CreateAzureBatchAccount creates a new azure batch account
+func CreateAzureBatchAccount(ctx context.Context, accountName, location, resourceGroupName string) (a batchARM.Account, err error) {
+	token, err := iam.GetResourceManagementToken(iam.AuthGrantType())
```

Comment:
On other samples we have preferred to move the client creation to its own function.